docs/docs/index.md

```diff
@@ -1,96 +1,148 @@
 # Overview
 
-Dataview is a live index and query engine over your knowledge base. You can associate *data* (like tags, dates,
-snippets, numbers, and so on) with your markdown pages, and then *query* (like filter, sort, transform) this data. This
-is a simple but powerful idea:
+Dataview is a live index and query engine over your personal knowledge base. You can [**add metadata**](annotation/add-metadata.md) to your notes and **query** them with the [**Dataview Query Language**](queries/structure.md) to list, filter, sort or group your data. Dataview keeps your queries always up to date and makes data aggregation a breeze.
 
-- Track sleep schedules and habits by recording them in daily notes, and automatically create weekly tables of your
-  sleep schedule.
+You could
+
+- Track your sleep by recording it in daily notes, and automatically create weekly tables of your sleep schedule.
 - Automatically collect links to books in your notes, and render them all sorted by rating.
-- Automatically collect pages annotated with a given date, showing them in your daily note or elsewhere.
+- Automatically collect pages associated with today's date and show them in your daily note.
 - Find pages with no tags for follow-up, or show pretty views of specifically-tagged pages.
-- Create dynamic views which show upcoming birthdays or events, annotated with notes.
+- Create dynamic views which show upcoming birthdays or events recorded in your notes
+
+and many, many things more.
+
+!!! hint "Dataview gives you a fast way to search, display and operate on indexed data in your vault!"
+
+Dataview is highly generic and high performance, scaling up to hundreds of thousands of annotated notes without issue. 
```

Comment:
IMO this could be simplified: "is high performance and scales to thousands of notes without issue."

Reply:
"is high performance" sounds odd to me. Shouldn't it be something like "Dataview is highly performant"?

Reply:
You are correct. Good catch.